### PR TITLE
Fix fixed table header for sticky

### LINKED ISSUE #5646: Sticky header broken in Bootstrap grids
**Bootstraptable version(s) affected**:

latest? (used the demo)

**Description**  

When placing a table with sticky header in a Bootstrap grid, it takes the full width.
Expected behaviour is that the sticky header stays within the grid area.

**Example**  

https://live.bootstrap-table.com/code/saitho/6954

### PR DESCRIPTION
**Pull request by ko22009**
Fix issue.

Fixes #5646

### CODE REVIEW

**Comment by djhvscf** [comment]
Please provide an [Online Example](https://live.bootstrap-table.com/) to show this PR working, thanks!

**Comment by ko22009** [comment]
@djhvscf, https://live.bootstrap-table.com/code/ko22009/6995

**Comment by ko22009** [comment]
@djhvscf, let see my example. I pushed another way realized by getBoundingClientRect

File: src/extensions/sticky-header/bootstrap-table-sticky-header.js [hunk 1]
```diff
@@ -113,16 +113,26 @@ $.BootstrapTable = class extends $.BootstrapTable {
...
       // stick it in position
+      const coords = this.$tableBody[0].getBoundingClientRect()
+      let width = '100%'
       let stickyHeaderOffsetLeft = this.options.stickyHeaderOffsetLeft
       let stickyHeaderOffsetRight = this.options.stickyHeaderOffsetRight
 
+      if (!stickyHeaderOffsetLeft) {
+        stickyHeaderOffsetLeft = coords.left
+      }
+      if (!stickyHeaderOffsetRight) {
```
Comment (UtechtDustin) [inline src/extensions/sticky-header/bootstrap-table-sticky-header.js:124]: elseif

**Comment by UtechtDustin** [comment]
Looks good for me, what do you think @djhvscf ?

**Review by UtechtDustin — APPROVED** [review]

**Review by djhvscf — APPROVED** [review]

**Pull request merged**